beacon-chain/blockchain/core.go

```diff
@@ -49,11 +49,29 @@ func NewBeaconChain(db ethdb.Database) (*BeaconChain, error) {
 	if err != nil {
 		return nil, err
 	}
+	hasGenesis, err := db.Has([]byte("genesis"))
+	if err != nil {
+		return nil, err
+	}
+	if !hasGenesis {
+		genesisBlock, err := types.NewGenesisBlock()
```

Comment:
Add a `log.Info` for something like `"No genesis block found on disk, initializing genesis block"`